tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -492,36 +493,41 @@ void test_sphere() {
 
   const std::array<
       std::variant<creators::Sphere::Excision, creators::Sphere::InnerCube>, 3>
-      interiors{{creators::Sphere::Excision{}, creators::Sphere::InnerCube{0.0},
-                 creators::Sphere::InnerCube{0.7}}};
+      interiors{{creators::Sphere::InnerCube{0.0},
+                 creators::Sphere::InnerCube{0.7},
+                 creators::Sphere::Excision{}}};
   const std::array<
       std::optional<creators::Sphere::EquatorialCompressionOptions>, 2>
       equatorial_compressions{
           {std::nullopt,
            creators::Sphere::EquatorialCompressionOptions{0.5, 2}}};
-  const std::array<std::vector<double>, 2> radial_partitioning{
-      {{}, {0.5 * (inner_radius + outer_radius)}}};
-  const std::array<std::vector<domain::CoordinateMaps::Distribution>, 2>
+  const double outer_minus_inner = outer_radius - inner_radius;
+  const std::array<std::vector<double>, 3> radial_partitioning{
+      {{},
+       {0.5 * (inner_radius + outer_radius)},
+       {inner_radius + 0.333 * outer_minus_inner,
+        inner_radius + 0.666 * outer_minus_inner}}};
```

Comment:
If these are supposed to be thirds make them actually thirds, otherwise make them not look like thirds.